.github/actions/file-diff/index.js

```diff
@@ -110,23 +110,26 @@ async function run() {
 				const md = ["", `#### ${name}`, ""];
 				const data = [name, bytesToSize(totalDiffSize)];
 
+				if (totalDiffSize - totalSize === 0) return;
+
 				if (hasDiff) {
 					// If a diff path was provided and the component folder doesn't exist,
 					// report that the compiled assets were removed
 					if (
 						!existsSync(join(diffPath, "components", name)) ||
 						(totalSize === 0 && totalDiffSize > 0)
 					) {
-						data.push("🚨 package deleted/moved/renamed");
+						data.push("🚨 deleted, moved, or renamed");
+						summaryTable.push(data);
 					} else if (totalSize > 0 && totalDiffSize === 0) {
-						data.push("🎉 new package");
-					} else {
+						data.push("🎉 new");
+						summaryTable.push(data);
+					} else if (bytesToSize(Math.abs(totalDiffSize - totalSize)) !== "< 0.01 KB") {
 						data.push(printChange(totalDiffSize - totalSize));
+						summaryTable.push(data);
 					}
 				}
 
-				summaryTable.push(data);
```

Comment:
@mdt2 I moved this into the if statements to prevent items being added to the summary table if they don't have any relevant changes.